### DISCUSSION TITLE
Problem on Altervista.org

### DISCUSSION THREAD

**Discussion by WECKLCIRO** [prasathmani/tinyfilemanager#694] [Q&A]
Hello guys, I downloaded the latest release of tinyfilemanager and I put it on the root folder of my website on Altervista.org but when I open tinyfilemanager.php I'm getting "Root path "/var/www/html" not found!" error. I already tried to change root path variable with a new folder that I created with 777 file permissions, but I'm still getting the same error("Root path "/path/to/folder" not found!"). How come? I'm using php 8 version and I already changed use-auth variable value to true. Thank you.

**Comment by prasathmani** [comment]
share your `$root_path` and `$root_url` config, so that I can help you.

**Reply by WECKLCIRO** [reply]
$root_path= '/wp-content/uploads/folder
$root_url= 'https://sitename.altervista.org'
I also tried $root_url= 'https://site name.altervista.org/wp-content/uploads' but I'm getting the same error.

**Reply by baur** [reply]
In my case works with this params:

```php
$root_path = $_SERVER['DOCUMENT_ROOT'].'/fm/data';
$root_url = '/fm/data';
```

**Reply by WECKLCIRO** [reply]
I've tried that and I get the same error "Root path "/var/www/html/Uploads/Files" not found!".

**Reply by baur** [reply]
where placed file **tinyfilemanager.php**?
`/var/www/html?`
then 
```php
$root_path = $_SERVER['DOCUMENT_ROOT'].'/Uploads/Files';
$root_url = '/Uploads/Files';
```

**Reply by WECKLCIRO** [reply]
I placed tinyfilemanager file in a new folder in the root. 
Yes, I put that!

**Reply by prasathmani** [reply]
```php
$root_path = $_SERVER['DOCUMENT_ROOT'].'/wp-content/Uploads/Files';
$root_url = 'wp-content/Uploads/Files';
```

**Reply by WECKLCIRO** [reply]
It's the same... on the previous message I tried to create a new uploads folder in the root and I still got the same error.

**Comment by Giglium** [comment]
The problem is that on Altervista `$_SERVER['DOCUMENT_ROOT']` returns `/var/www/html`, which doesn't work. The real path is `/membri/yourusername/` (More info here: https://forum.en.altervista.org/services/173-root-path.html).

I placed my file in the Altervista root (not inside a subfolder), then the fix for me was changing the `$root_path`:
```php
$root_path = '/membri/yourusername';
```

> Replace `yourusername` with your actual Altervista username (the one in your `yourusername.altervista.org` URL).
